c2corg_ui/static/js/documentediting.js

```diff
@@ -196,7 +199,7 @@ app.DocumentEditingController.prototype.submitForm = function(isValid) {
   }
 
   // push to API
-  var data = this.scope_[this.modelName_];
+  var data = angular.copy(this.scope_[this.modelName_]);
```

Comment:
It seems it is not equivalent.
``goog.object.clone`` does not prevent the geometry field of the model from being updated when changing it in the copy. Maybe because it is only a flat clone:
https://closure-library.googlecode.com/git-history/docs/namespace_goog_object.html
whereas ``angular.copy`` is a deep copy:
https://docs.angularjs.org/api/ng/function/angular.copy